### DISCUSSION TITLE
Show more info for `git update-git-for-windows`

### DISCUSSION THREAD

**Discussion by goyalyashpal** [git-for-windows/git#4839] [Ideas]
on trying to update from gfw v2.39.1, it showed:

```console
$ git update-git-for-windows   
Git for Windows 2.39.1.windows.1 (64-bit)
Update 2.44.0.windows.1 is available
Download and install Git for Windows v2.44.0.windows.1 [N/y]? y
############################################ 100.0%

```

* it should show the update download size
as someone maybe on a limited internet bandwidth or any such reason.

* if possible then it should also show where that download is getting stored. 
this might not be wanted generally, but i just thought to share the suggestion.

the reason for show download location is that in case the immediately following install fails, then user can retry perself. as it happened with me where I:

* invoked the command `git update-git-for-windows` via msys2 `bash` on `konsole` terminal i.e. in user mode
* the install location requires admin permissions 
* so the install failed (even though, i guess i filled the password)

### Versions:

```console
$ git -v
git version 2.39.1.windows.1

$ systeminfo | grep -iA1 "os name"
OS Name:                   Microsoft Windows 10 Home Single Language
OS Version:                10.0.19045 N/A Build 19045

```

**Comment by dscho** [comment]
The source code of that script is stored here: https://github.com/git-for-windows/build-extra/blob/HEAD/git-extra/git-update-git-for-windows. Please make the proposed changes and open a PR.

**Reply by goyalyashpal** [reply]
thanks a lot for sharing the src location. i will see what i can do 😃 
but don't hold ur breadth 😅

**Reply by goyalyashpal** [reply]
oh, and what will be ur thoughts on showing the download location? does that make sense or nobody cares?

**Reply by dscho** [reply]
> thoughts on showing the download location?

Wouldn't hurt 😁
